freqtrade/freqai/data_kitchen.py

```diff
@@ -724,6 +724,113 @@ def rotate_point(origin, point, angle):
 
         return
 
+    def compute_inlier_metric(self, set_='train') -> None:
+        """
+
+        Compute inlier metric from backwards distance distributions.
+        This metric defines how well features from a timepoint fit
+        into previous timepoints.
+        """
+
+        import scipy.stats as ss
```

Comment:
For (world-) historic reasons (some people are sensitive to this) - i think we should pick a different alias (anything other than `ss` will work fine).

the most obvious one might be `stats` - which would allow you to rewrite the import to `from scipy import stats` (flake will tell you about that, too) :laughing:

Reply:
I agree!